src/features/ShowHelp.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import vscode = require("vscode");
+import { LanguageClient, NotificationType, RequestType } from "vscode-languageclient";
+import { IFeature } from "../feature";
+
+export const ShowOnlineHelpRequestType =
+    new RequestType<string, void, void, void>("powerShell/showOnlineHelp");
+    // I think we can take out the ShowOnlineHelpRequestType... I don't think we actually send it...
+export const ShowHelpRequestType =
+    new RequestType<string, void, void, void>("powerShell/showHelp");
+export class ShowHelpFeature implements IFeature {
+
+    private command: vscode.Disposable;
+    private deprecatedCommand: vscode.Disposable;
+    // TODO: Remove Dummy command before merge.
+    // Dummy command is used to test out the write-warning in PSES
+    // when you send powerShell/showOnlineHelp instead of powerShell/showHelp.
+    private dummyCommand: vscode.Disposable;
+    private languageClient: LanguageClient;
+
+    constructor() {
+        // TODO: Remove this before merge.
```

Comment:
what's this `TODO` for? To delete the registerCommand?

Reply:
Yeah, for that commit I wanted to add a way to send the message to PSES, the TODOs are a reminder to remove them before merging the pull request. I mostly wanted it there in case someone wanted to test it to see what it's behavior would be.